tests/sqllogictests/suites/base/05_ddl/05_0000_ddl_create_tables.test

```diff
@@ -141,7 +141,7 @@ statement error 1006
 CREATE TABLE db2.test5(a Varchar null, y Varchar null) ENGINE=fuse AS SELECT a, b, a FROM db1.test1
 
 statement ok
-create table db2.test6(id Int8, created timestamp  DEFAULT CURRENT_TIMESTAMP)
+create table db2.test55(id Int8, created timestamp  DEFAULT CURRENT_TIMESTAMP)
```

Comment:
don't forget to drop it

Reply:
in this ut, it will `DROP DATABASE db2` after all cases passed, and table `test6` also has no single drop statement.